package.json

```diff
@@ -28,18 +29,18 @@
     "param-case": "^2.1.1"
   },
   "peerDependencies": {
-    "bedrock": "1.15.0 - 3.x",
-    "bedrock-express": "2.0.6 - 3.x",
+    "bedrock": "1.x - 3.x",
+    "bedrock-express": "2.x - 3.x",
     "bedrock-ledger-context": "^15.0.0",
     "bedrock-ledger-node": "8.x - 10.x",
-    "bedrock-mongodb": "5.x - 6.x",
-    "bedrock-passport": "3.x - 5.x",
-    "bedrock-permission": "^2.3.2",
+    "bedrock-mongodb": "7.x - 8.x",
+    "bedrock-passport": "3.x - 6.x",
+    "bedrock-permission": "^3.0.0",
     "bedrock-rest": "2.1.0 - 3.x",
     "bedrock-server": "^2.2.0"
   },
   "engines": {
-    "node": ">=8"
+    "node": ">=10"
```

Comment:
@mattcollier I believe this needs to be node 12 now.

Reply:
I think at this point in time we would just change it to 14.

Reply:
Ok I will do that after your pending PR is merged into this one. I dogwood aware of your code coming into this branch?